storage/beacondb/eth1.go

```diff
@@ -31,7 +34,7 @@ func (kv *KVStore) GetLatestExecutionPayloadHeader() (
 	if err != nil {
 		return nil, err
 	}
-	kv.latestExecutionPayloadCodec.SetActiveForkVersion(forkVersion)
+	kv.latestExecutionPayloadCodec.SetActiveForkVersion(bytes.FromUint32(forkVersion))
```

Comment:
this is so bad, but indeed it requires a coordinate update to change the way we serialize data

Reply:
Yeah annoying, but erred on the side of keeping state consistent. So have to deal with a manual conversion before storage :/